File: Max Consecutive Ones/Max consecutive ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int ans = 0;
        int count = 0;
        for(int i = 0; i < nums.size(); i++) {
            if (nums[i] == 1) {
                count++;
                if (count > ans) {
                    ans = count;
                }
            }
            else {
                count = 0;
            }
        }
        
        return ans;
    }
};

File: Max Consecutive Ones/Max consecutive ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        ans = 0
        count = 0
        for num in nums:
            if (num == 1):
                count += 1
                if (count > ans):
                    ans = count
            else:
                count = 0
        return ans